Makefile

```diff
@@ -26,7 +26,7 @@ $(APP_EXE): app/*.go render/*.go report/*.go xfer/*.go
 $(PROBE_EXE): probe/*.go probe/docker/*.go probe/endpoint/*.go probe/host/*.go probe/process/*.go probe/overlay/*.go report/*.go xfer/*.go
 
 $(APP_EXE) $(PROBE_EXE):
-	go get -tags netgo ./$(@D)
+	go get -d -tags netgo ./$(@D)
```

Comment:
Please correct me if I'm wrong, but as I understand it, the `go get` step here has the purpose of fetching the dependencies for the component. But without -d, it also builds and installs the component itself to $GOPATH/bin — needless work that we duplicate on the line below.